Sources/TuistKit/Generator/GeneratorModelLoader.swift

```diff
@@ -242,12 +276,24 @@ extension TuistGenerator.Project {
     func adding(target: TuistGenerator.Target) -> TuistGenerator.Project {
```

Comment:
Given that `TuistGenerator.Project` is a class, we don't need to be creating copies of the objects. Wherever `adding` and `replacing` is used, I'd just set the attribute to the object and not create new instances of it.

Reply:
I believe this is related to my previous comment about `Project`'s properties being immutable.